.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Tracking feature
+about: Use this template for tracking new features.
+title: "[DATE]: [FEATURE NAME]"
+labels: tracking feature, needs planning
+assignees: wwelling
+---
+
+1. Purpose
+    > What business use case is this for?
+
+
+2. Scope
+    > What is the scope of the feature?
+
+
+3. Overview
+    > Please provide as much details as necessary to describe the feature.
+
+
+4. Reference Material
+    > Please provide only links that are reputable fully qualified domain names.
```

Comment:
```suggestion
    > Please provide links.
```

I think we should drop this part: "that are reputable fully qualified domain names.".

I think this is more sensible: "Please provide links.".